inference-engine/tests/functional/plugin/cpu/single_layer_tests/psroi_pooling.cpp

```diff
@@ -0,0 +1,188 @@
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<ROIAlignLayerCPUTestParamsSet> obj) {
+        CPULayerTestsDefinitions::PSROIPoolingLayerTestParams basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = obj.param;
+        std::string td;
+        Precision netPr;
+        PSROIPoolingSpecificParams psroiPar;
+        std::tie(psroiPar, netPr, td) = basicParamsSet;
+        std::tie(featureMapShape, proposal, outputDim, groupSize,
+                 spatialScale, spatialBinsX, spatialBinsY, mode) = psroiPar;
+        std::ostringstream result;
+        result << "ROIAlignTest_";
+        result << std::to_string(obj.index) << "_";
+        result << "binsX=" << spatialBinsX << "_";
+        result << "binsY=" << spatialBinsY << "_";
+        result << "spatialScale=" << spatialScale << "_";
+        result << "outputD=" << outputDim << "_";
+        result << "groupS=" << groupSize << "_";
+        result << (netPr == Precision::FP32 ? "FP32" : "BF16") << "_";
+        result << mode << "_";
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+        return result.str();
+    }
+protected:
+    void SetUp() override {
+        CPULayerTestsDefinitions::PSROIPoolingLayerTestParams basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+        CPULayerTestsDefinitions::PSROIPoolingSpecificParams psroiPoolingParams;
+        auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
+        std::tie(psroiPoolingParams, netPrecision, targetDevice) = basicParamsSet;
+        inPrc = outPrc = netPrecision;
+        std::tie(featureMapShape, proposal, outputDim, groupSize,
+                 spatialScale, spatialBinsX, spatialBinsY, mode) = psroiPoolingParams;
+
+
+        ngraph::Shape proposalShape = { proposal.size() / 5, 5 };
+
+        auto coords = ngraph::builder::makeConstant<float>(ngraph::element::f32, proposalShape, proposal);
+        auto params = ngraph::builder::makeParams(ngraph::element::f32, {featureMapShape});
+
+        auto psroi = std::make_shared<ngraph::op::v0::PSROIPooling>(params[0], coords, outputDim, groupSize,
+                                                       spatialScale, spatialBinsX, spatialBinsY, mode);
+        psroi->get_rt_info() = getCPUInfo();
+        selectedType = std::string("unknown_") + inPrc.name();
+
+        threshold = 0.001f;
+        const ngraph::ResultVector results{std::make_shared<ngraph::opset3::Result>(psroi)};
+        function = std::make_shared<ngraph::Function>(results, params, "PSROIPooling");
+    }
+};
+
+TEST_P(PSROIPoolingLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    Run();
+    CheckCPUImpl(executableNetwork, "PSROIPooling");
+}
+
+namespace {
+
+/* CPU PARAMS */
+std::vector<CPUSpecificParams> filterCPUInfoForDevice() {
```

Comment:
Could you use `filterCPUSpecificParams` from "cpu_test_utils.hpp"?

Reply:
Done